packages/client/src/views/CorrectionForm/utils.ts

```diff
@@ -98,6 +98,23 @@ export function isCorrection(declaration: IDeclaration) {
   )
 }
 
+export function isFileSizeExceeded(declaration: IDeclaration) {
+  const {
+    data: { documents }
+  } = declaration
+  let totalFileSize = 0
+  for (const index in documents) {
+    if (!isArray(documents[index])) {
+      continue
+    }
+    ;(documents[index] as IFileValue[]).forEach((fieldValue) => {
+      totalFileSize += fieldValue.fileSize || 0
+    })
+  }
+
+  return totalFileSize > 20480000
```

Comment:
I noticed that this isn't using the `ACCUMULATED_FILE_SIZE` variable. Should it?

Reply:
Yes, it should. @yeasinhossain-dsi if you are putting ACCUMULATED_FILE_SIZE in window.config, put it in bytes, use it here and in a human readable string, convert it to 20MB based on byte size.  Adding ACCUMULATED_FILE_SIZE in config is a nice feature that I didnt think about, but if you add it then we need to use it everywhere.  Currently there is no request for this limit to be configurable.  So you could just use a content key with no config variable and the user just translates 20MB in the content key.  May be easier and quicker at this stage than to persist with this config variable